tfjs-backend-webgpu/src/index.ts

```diff
@@ -52,6 +62,7 @@ if (device_util.isBrowser() && isWebGPUSupported()) {
           `it doesn't support synchronously to read data from GPU.`);
     }
     const device: GPUDevice = await adapter.requestDevice(deviceDescriptor);
+    setDeviceLimits(device.limits);
```

Comment:
Can you do a little refactor here for WebGPUBackend which only accept an optional parameter `device` and move above logic to the constructor of `WebGPUBackend` if no `device` is passed? And do `setDeviceLimits` in `WebGPUBackend` rather than before a new `WebGPUBackend` generated.

Reply:
`setDeviceLimits` moved to constructor of WebGPUBackend, PTAL